### PR TITLE
Fix/san 5100

### PR DESCRIPTION
**Pull request by GarageInc**
0. Add expand button for chart
image

1. Fixed default styles in mobiles for triggers list
image

image
2. Fixed triggers creation and form validation
image

3. Adding 1d min value for cooldown with type "DAA"
image

### CODE REVIEW

**Comment by IvanIvanoff** [comment]
The wording `Once per` is not precisely correct. It should have the `No more than once per` meaning (probably worded differently)

File: src/pages/SonarFeed/SonarFeedPage.module.scss [hunk 1]
```diff
@@ -1,35 +1,48 @@
...
-    color: var(--white) !important;
-    fill: var(--white) !important;
-
-    &__icon {
-        height: 18px;
-        margin-right: 7px;
-    }
+  background: var(--green) !important; // NOTE(vanguard): Delete after updating ui button green variant
+  color: var(--white) !important;
+  fill: var(--white) !important;
```
Comment (DmitriVanGuard) [inline src/pages/SonarFeed/SonarFeedPage.module.scss]: This css is no longer necessary. `.newSignal` button could be defined with following props: `fill`, `accent='positive'`

File: src/pages/SonarFeed/SonarFeedPage.module.scss [hunk 2]
```diff
@@ -1,35 +1,48 @@
...
+    height: 18px;
+    margin-right: 7px;
+  }
 }
 
 .search,
-.filter{
-    fill: var(--black);
+.filter {
+  fill: var(--black);
```
Comment (DmitriVanGuard) [inline src/pages/SonarFeed/SonarFeedPage.module.scss]: Same as the `[D.1]` comment

File: src/pages/SonarFeed/SonarFeedPage.js [hunk 3]
```diff
@@ -71,45 +71,47 @@ const SonarFeed = ({ location: { pathname }, isLoggedIn, match }) => {
...
+            component={Loadable({
+              loader: () => import('./SignalDetails'),
+              loading: () => <PageLoader />
+            })}
+          />
+          ,
+          <Route
+            path={detailsModalLocation}
+            exact
+            component={Loadable({
```
Comment (DmitriVanGuard) [inline src/pages/SonarFeed/SonarFeedPage.js]: Better to move `Lodable`s to a constant outside component render

File: src/ducks/Signals/signalFormManager/SignalMaster.js [hunk 4]
```diff
@@ -173,6 +176,7 @@ export class SignalMaster extends React.PureComponent {
 
 const mapDispatchToProps = dispatch => ({
   createTrigger: payload => {
+    console.log(JSON.stringify(payload))
```
[the remark this answers is not in the document]
Comment (GarageInc) [reply src/ducks/Signals/signalFormManager/SignalMaster.js]: Removed

File: src/ducks/Signals/signalFormManager/signalCrudForm/formParts/TriggerFormMetricValues.js [hunk 5]
```diff
@@ -84,7 +85,7 @@ export const TriggerFormMetricValues = ({
           'walletBalanceChangeType'
         ) && (
         <div className={styles.Field}>
-          <label>Condition</label>
+          <div className={styles.label}>Condition</div>
```
[the remark this answers is not in the document]
Comment (GarageInc) [reply src/ducks/Signals/signalFormManager/signalCrudForm/formParts/TriggerFormMetricValues.js]: Thx!

File: src/pages/SonarFeed/SonarFeedActivityPage.module.scss [hunk 6]
```diff
@@ -1,16 +1,21 @@
 @import "~@santiment-network/ui/mixins.scss";
 
 .wrapper {
-    margin: 0 auto;
-    max-width: 720px;
+  margin: 0 auto;
+  max-width: 720px;
+
+  img {
```
[the remark this answers is not in the document]
Comment (GarageInc) [reply src/pages/SonarFeed/SonarFeedActivityPage.module.scss]: No :( It is markdown from server.

File: src/pages/SonarFeed/SonarFeedPage.js (hunk 3 above)
Comment (GarageInc) [reply src/pages/SonarFeed/SonarFeedPage.js]: What is you mean? Create a som builder function, which will return Loadable?

Comment (DmitriVanGuard) [reply src/pages/SonarFeed/SonarFeedPage.js]: [Example](https://github.com/santiment/app/blob/2de507ec9b90fbc5cb20c4586e6b7afa41581386/src/App.js#L55)

Comment (GarageInc) [reply src/pages/SonarFeed/SonarFeedPage.js]: Thanks! Will do

Comment (GarageInc) [reply src/pages/SonarFeed/SonarFeedPage.js]: Done

File: src/pages/SonarFeed/SonarFeedPage.js [hunk 7]
```diff
@@ -89,25 +99,18 @@ const SonarFeed = ({ location: { pathname }, isLoggedIn, match }) => {
           <Route
             path={`${baseLocation}/details/:id`}
             exact
-            component={Loadable({
-              loader: () => import('./SignalDetails'),
-              loading: () => <PageLoader />
-            })}
+            render={props => <LoadableSignalDetailsPage {...props} />}
```
Comment (DmitriVanGuard) [inline src/pages/SonarFeed/SonarFeedPage.js]: In such case, when we don't need to pass specific props to a renderable route, we can pass desired component to a `Route` via `component` props. e.g: `component={LoadableSignalDetailsPage}`

**Pull request merged**